src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -41,6 +46,36 @@ public static partial class JsonSerializer
                 throw new ArgumentNullException(nameof(json));
             }
 
+            return Deserialize<TValue>(json.AsSpan(), typeof(TValue), options);
+        }
+
+        /// <summary>
+        /// Parse the text representing a single JSON value into an instance of a specified type.
+        /// </summary>
+        /// <returns>A <typeparamref name="TValue"/> representation of the JSON value.</returns>
+        /// <param name="json">JSON text to parse.</param>
+        /// <param name="options">Options to control the behavior during parsing.</param>
+        /// <exception cref="JsonException">
+        /// The JSON is invalid.
+        ///
+        /// -or-
+        ///
+        /// <typeparamref name="TValue" /> is not compatible with the JSON.
+        ///
+        /// -or-
+        ///
+        /// There is remaining data in the span beyond a single JSON value.</exception>
+        /// <exception cref="NotSupportedException">
+        /// There is no compatible <see cref="System.Text.Json.Serialization.JsonConverter"/>
+        /// for <typeparamref name="TValue"/> or its serializable members.
```

Comment:
@layomia, @tdykstra, can we please add `or its serializable members` to the docs to all such methods, in the published api-docs?

It doesn't show that currently. For example:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserialize?view=netcore-3.1#System_Text_Json_JsonSerializer_Deserialize_System_String_System_Type_System_Text_Json_JsonSerializerOptions_

https://github.com/dotnet/dotnet-api-docs/blob/fcf78e35bc4eca5734d3f248670b1ade805e2ed0/xml/System.Text.Json/JsonSerializer.xml#L86